### PR TITLE
[2843] Add SEND to subject filter

### PR DESCRIPTION
**Pull request by timabell**
### Context
```
As a user
I need/want to filter my search by whether or not the subject is a SEND course
So that I can find SEND courses
```

### Changes proposed in this pull request

Add the accordion Special educational needs and disabilities (SEND) with a single checkbox: 'Show only courses with a SEND specialism' that passes the `senCourses` parameter as true/false.

<kbd>
send
</kbd>

### Guidance to review

:ship:

* https://s121d02-2843-find-as.azurewebsites.net/results/filter/subject?qualifications=QtsOnly,PgdePgceWithQts,Other&fulltime=False&parttime=False&hasvacancies=True&senCourses=True

### CODE REVIEW

**Comment by timabell** [comment]
[[S] Add 'is send' checkbox to subjects page](https://trello.com/c/wXHyFiYe/2843-s-add-is-send-checkbox-to-subjects-page)

File: spec/features/result_filters/subjects_spec.rb [hunk 1]
```diff
@@ -36,6 +37,10 @@
       subject_filter_page.subject_areas.second.then do |subject_area|
         expect(subject_area.name.text).to eq("Secondary")
       end
+
+      subject_filter_page.send_area.then do |subject_area|
+        expect(subject_area.name.text).to eq("Special educational needs and disability (SEND)")
+      end
```
Comment (faissalMT) [inline spec/features/result_filters/subjects_spec.rb]: Don't worry about testing for the presence of static text

**Comment by defong** [comment]
image

General eye candies 
- [ ] Subject/Send bold
- [ ] Bursaries/Scholarship/Early career payments
- [ ] SKE
- [ ] A-Z ordering for subjects by their subject name
- [ ] Categories headings (Secondary: Modern languages vs Secondary: Modern `L`anguages)
- [ ] Categories headings (Further education vs Further `E`ducation )

File: spec/site_prism/page_objects/page/result_filters/subject_page.rb [hunk 2]
```diff
@@ -15,6 +15,7 @@ class SubjectAreaSection < SitePrism::Section
       class SubjectPage < SitePrism::Page
         set_url "/results/filter/subject{?query*}"
         sections :subject_areas, SubjectAreaSection, '[data-qa="subject_area"]'
+        section :send_area, SubjectAreaSection, '[data-qa="send_area"]'
```
Comment (faissalMT) [inline spec/site_prism/page_objects/page/result_filters/subject_page.rb]: I question whether or not SEND is a subject area from a data modelling perspective, but this should be fine for now

**Pull request merged**